### PR TITLE
Adding basic UI for twitch extension

### PR DESCRIPTION
**Pull request by lucasmontano**

### CODE REVIEW

File: viewer.js [hunk 1]
```diff
@@ -7,21 +7,21 @@ var twitch = window.Twitch.ext
...
+    url: "https://montano-twitch-extension.herokuapp.com/",
     success: renderTopParticipants,
     error: logError,
   }
 }
 
 function setAuth(token) {
   Object.keys(requests).forEach((req) => {
-    requests[req].headers = { Authorization: `Bearer ${token}` }
+    //requests[req].headers = { Authorization: `Bearer ${token}` }
```
Comment (omurilo) [inline viewer.js]: Did you leave the setAuth function because you plan to use it in the future?

I think if you don't use it, you can remove it.

Comment (jasondavindev) [inline viewer.js]: Add TODO tag

Comment (jasondavindev) [reply viewer.js]: No comments in unused codes

**Review by omurilo — APPROVED** [review]
Thankfully we can better understand the doc from the twitch ext, right, finally working!

Congratulations!

**Review by jasondavindev — APPROVED** [review]
I suggest replace `var` by `let` and `const` for follow Javascript standards

File: viewer.js (hunk 1 above)
Comment (omurilo) [reply viewer.js]: You too can remove node_modules, package.json, package-lock.json....

All this files to be EBS, so could remove it.

**Review by omurilo — APPROVED** [review]

**Pull request merged**